tools/vccexe/vccdiscover.nim

```diff
@@ -0,0 +1,98 @@
+## VCC compiler backend discovery Utility
+##
+## Module to discover the path to the vcvarsall utility of a VCC compiler backend.
+## The module supports discovery for either the latest recognizable version (default)
+## or discovery of a specific VCC compiler backend version.
+##
+## This module can also produce a standalone command-line executable.
```

Comment:
Hmmm, in case you just want the path to your Visual Studio Compiler, or you want to check whether it actually finds the right installed version (if you have multiple installed).

But yes, we do not really need the `vccdiscover` to be a command-line executable. But hey, if we never compile it on its own, that code never gets used until the day we actually need it, or someone wants to see how to use the module. If that never happens, the additional code starting from [Line 41](https://github.com/couven92/Nim/blob/devel/tools/vccexe/vccdiscover.nim#L41) simply won't do any harm...

But if you really want it gone, I can delete it. Then we do not need to maintain it in the future...